app/controllers/contents_controller.rb

```diff
@@ -3,14 +3,22 @@ class ContentsController < ApplicationController
 
   # GET /contents
   def index
-    @contents = Content.all
-
-    render json: @contents
+    render json: {
+      contents: Content.all,
+      content_metadata: ContentMetadatum.all,
+      content_audios: ContentAudio.all,
+      content_resources: ContentResource.all
```

Comment:
Does this mean metadata, audio and resources are included in the response in a GET request to `/contents`? If so, the response will be very large and hard to parse, thus I think we shall keep the default behaviour and not include any metadata,audio or resources. That information can be request separately with a GET request to `/contents/<id>`

Reply:
Seeing as the same method is called for contents, metadata, audios and resources, I would assume this is the case. I'll revert the code back to the default behavior as requested.

Reply:
👍